src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.discussions;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+/**
+ * Created by qaga on 2015-11-25.
+ */
+public class PostsListPage extends WikiBasePageObject{
+
+  @FindBy(css = ".post-detail")
+  private List<WebElement> postList;
+
+  private static final String PATH = "d/f/%s";
+  private static final String DEFAULT_ID = "203236";
+
+  public PostsListPage(WebDriver driver) {
+    super(driver);
+  }
+
+  public PostsListPage open(String wikiID) {
+    driver.get(urlBuilder.getUrlForWiki().replace("/wiki", "") + String.format(PATH, wikiID));
+    return this;
+  }
+
+  public PostsListPage open() {
+    return open(DEFAULT_ID);
+  }
+
+  public boolean isPostListNotEmpty() {
+    return postList.size() > 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)